tests/acceptance/features/apiSpaces/listSpaces.feature

```diff
@@ -131,3 +131,23 @@ Feature: List and create spaces
       | name             | Project Venus                    |
       | quota@@@total    | 2000                             |
       | root@@@webDavUrl | %base_url%/dav/spaces/%space_id% |
+
+  
+  Scenario: The owner of the project space should be who created space or manager if the creater leaved project space
```

Comment:
```suggestion
  Scenario: The project space owner should always be one of the space managers
```

Reply:
Then I misunderstood and should have added another case 
Case:
- user1 creates space
- user1 adds user2 as manager
- user2 changes user1s role as editor

How is owner?
I thought that user1.

Reply:
your described scenario is correct.... the implementation (in my pr https://github.com/cs3org/reva/pull/3583) picks the first user match from the sum of managers. So we cannot guarantee any order here. Thats not the best solution but is working for the moment.

a space type project in theory, should never be owned by someone, it can have a creator but not a concrete owner... it also can have managers, but not a owner.